src/loose/oose/fis/lab/student/manager/controllers/LoginController.java

```diff
@@ -0,0 +1,79 @@
+package loose.oose.fis.lab.student.manager.controllers;
+
+
+import javafx.fxml.FXML;
+
+import javafx.scene.control.PasswordField;
+
+import javafx.scene.control.TextField;
+
+import javafx.scene.text.Text;
+
+
+public class LoginController {
+
+
+    @FXML
+
+    public Text loginMessage;
+
+    @FXML
+
+    public PasswordField passwordField;
+
+    @FXML
+
+    public TextField usernameField;
+
+
+    @FXML
+
+    public void handleLoginButtonAction() {
+
+        String username = usernameField.getText();
+
+        String password = passwordField.getText();
+
+
+        if (username == null || username.isEmpty()) {
```

Comment:
If login as a teacher succeeds, the user should be taken to the view-students scene